Projects/Feature/OutingApplyFeature/Sources/OutingApplyView.swift

```diff
@@ -18,81 +18,147 @@ struct OutingApplyView: View {
+                                        }
                                     }
                                 }
+                                OutingApplyTextField("동행인", "동행인 추가 ﹢", $selectStudentsText, false)
+                                    .padding(.horizontal, 12)
+                                    .padding(.vertical, 12)
+                                    .disabled(true)
+                                    .onTapGesture {
+                                        viewModel.isShowingBottomSheet = true
+                                    }
                             }
+                            .padding(.top, 12)
 
-                            OutingApplyTextField("외출 장소", "외출 장소를 입력해주세요.", $placeText)
-                        }
-                        .padding(.bottom, 40)
-
-                        VStack(alignment: .leading) {
-                            Text("외출 사유")
-                                .dmsFont(.body(.body2), color: .GrayScale.gray10)
-                                .padding(.bottom, 8)
-
-                            DMSFormTextEditor("내용을 입력해주세요.", text: $reasonText, minHeight: 150)
-                        }
+                            VStack(alignment: .leading) {
+                                Text("외출 사유")
+                                    .dmsFont(.body(.body2), color: .GrayScale.gray10)
+                                    .padding(.bottom, 8)
 
-                        DMSWideButton(text: "외출 신청하기", color: .PrimaryVariant.primary) {
-                            viewModel.isShowingBottomSheet = true
+                                DMSFormTextEditor("내용을 입력해주세요.", text: $reasonText, minHeight: 150)
+                            }
+                            DMSWideButton(text: "외출 신청하기", color: .PrimaryVariant.primary) {
+                                viewModel.outingTypeTitleApplication = typeText
+                                viewModel.outingReasonApplication = reasonText
+                                viewModel.isPresentedOutingApplicationItemAlert = true
+                            }
+                            .alert("", isPresented: $viewModel.isPresentedOutingApplicationItemAlert) {
+                                Button("취소", role: .cancel) {}
+                                Button("확인",
+                                       role: .destructive,
+                                       action: {
+                                    if viewModel.outingApplicationTimeStatus && viewModel.outingTypeTitleApplicationStatus {
+                                        viewModel.confirmOutingApplicationItemButtonDidClicked()
+                                    } else {
+                                        viewModel.isErrorOcuured = true
+                                        viewModel.errorMessage = "필수 입력 항목을 확인해주세요."
+                                    }
+                                }
+                                )
```

Comment:
```suggestion
                                Button("확인", role: .destructive) {
                                    if viewModel.outingApplicationTimeStatus && viewModel.outingTypeTitleApplicationStatus {
                                        viewModel.confirmOutingApplicationItemButtonDidClicked()
                                    } else {
                                        viewModel.isErrorOcuured = true
                                        viewModel.errorMessage = "필수 입력 항목을 확인해주세요."
                                    }
                                }
```
아니면 아예 이렇게 가도 괜찮을거같아요